src/layout/VerifyEmail.component.js

```diff
@@ -9,6 +9,8 @@ const sendEmailVerificationMutation = {
     type: 'create',
 }
 
+const emailRegex = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/i
```

Comment:
this regex validation was gotten from the email validation from the [dhis2-ui library](https://github.dev/dhis2/ui/blob/69b3ec912bbe75cccef2640d2d7ab317bb117ae1/collections/forms/src/validators/email.js#L28-L29).